tensorrt_llm/_torch/pyexecutor/cuda_graph_runner.py

```
@@ -28,58 +29,60 @@ class CUDAGraphRunner:
+        self.dist = dist
+        self.kv_cache_manager_key = kv_cache_manager_key
+
+        # --- Internal state ---
         self.graphs: Dict[Tuple[int, int], torch.cuda.CUDAGraph] = {}
         self.static_inputs: Dict[Tuple[int, int], Dict[str, torch.Tensor]] = {}
         self.graph_outputs: Dict[Tuple[int, int],
                                  Callable[[], Optional[torch.Tensor]]] = {}
         self.graph_metadata: Dict[Tuple[int, int], Dict[str, Any]] = {}
-        self.memory_pool = engine._cuda_graph_mem_pool
+        self.memory_pool = cuda_graph_mem_pool
         self.padding_dummy_request: Optional["Request"] = None
```

Ruff (0.12.2)

Undefined name `Request`

(F821)